src/librustc_metadata/locator.rs

```diff
@@ -488,6 +488,8 @@ impl<'a> CrateLocator<'a> {
                 && self.triple != TargetTriple::from_triple(config::host_triple())
             {
                 err.note(&format!("the `{}` target may not be installed", self.triple));
+            } else if self.crate_name == sym::profiler_builtins {
+                err.note(&"the compiler may have been built without `profiler = true`");
```

Comment:
nit: let's not reference specific compiler configuration keys in an error message, I think "without profiler runtimes" (or something that maps to its comment in config.toml.example) should suffice.

Reply:
done